test/N3StreamParser-test.js

```diff
@@ -27,6 +27,12 @@ describe('N3StreamParser', function () {
 
     it('parses two triples', shouldParse(['<a> <b>', ' <c>. <d> <e> ', '<f>.'], 2));
 
+    it.skip('should parse double values that are split accross chunks in the stream', function (done) {
+      // this is the same as creating a read-stream from a file, with this one triple, where the
+      // highWaterMark of the readstream is set to 2.
+      shouldParse('<sub> <pred> 11.2 .'.match(/.{1,2}/g), 1)(done);
```

Comment:
I love the use of regex here by the way. Somehow forgot that `match` with `/g` returns an array of matches.